src/main/kotlin/org/openrewrite/kotlin/internal/PsiElementAssociations.kt

```diff
@@ -242,8 +242,8 @@ class PsiElementAssociations(val typeMapping: KotlinTypeMapping, val file: FirFi
                         }
                     }
 
-                    is FirArrayOfCall -> {
-                        // `FirArrayOfCall` is not a `FirFunctionCall`, so a `JavaType$Method` is impossible.
+                    is FirArrayLiteral -> {
+                        // `FirArrayLiteral` is not a `FirFunctionCall`, so a `JavaType$Method` is impossible.
```

Comment:
Is this empty branch still needed? I feel like this comment is not appropriate anymore 🤔

Reply:
Yes, this can be remove -- FirArrayLiteral clearly communicates it's not a method invocation, thank you!

Reply:
Redundant when branch removed